src/components/common/form/InputWithAdornment.js

```diff
@@ -18,6 +18,7 @@ const InputText = ({
   adornmentSize = 16,
   adornmentStyle,
   adornmentDisabled = false,
+  showError = true,
```

Comment:
why it's needed ? why not just {error & <ErrorText..... ?

Reply:
Because there should be white space by default when using text input in other places. 
For ReceiveToAddress screen it is not needed. so I removed it with flag

Reply:
Ok